src/cli.rs

```diff
@@ -159,6 +159,12 @@ Defaults to \"format\" if the NO_COLOR env is set and to \"none\" if stdout is n
     #[clap(short = 'd', long)]
     pub download: bool,
 
+    /// During download, keep the raw encoding of the body. Requires --download.
```

Comment:
It would be nice to use this option outside of download mode e.g to debug a server that returns an incorrect `content-encoding` header.

Can you disable the decompression in `Printer::print_response_body` when this flag is used?

Reply:
Good idea and thanks for the feedback. I believe I updated this as you wanted.